src/commands/community/nft/list.ts

```diff
@@ -139,18 +114,32 @@ async function renderSupportedNFTList(collectionList: NFTCollection[]) {
     ctx.fillText(collectionName, clt1stColConf.x, clt1stColConf.y)
 
     // chain name
+    const rectHeight = 40
+    const rectWidth = widthOf(ctx, chainName)
     const chainNameConf = {
       x: rowIn1stCol.x,
       y: clt1stColConf.y + clt1stColConf.mb,
       mb: 20,
     }
 
+    const col1stRectStats: RectangleStats = {
```

Comment:
2 same code blocks -> please generalize this
the only different thing between these 2 loops is `x`, please use loop index to adjust correct x
e.g. 
```
i % 2 === 0 -> x = 0
i % 2 === 1 -> x = 400
```